atomate/vasp/workflows/presets/core.py

```diff
@@ -53,6 +53,34 @@ def wf_bandstructure(structure, c=None):
 
     return wf
 
+def wf_scan_opt(structure,c={}):
+
+    c = c or {}
+    vasp_cmd = c.get("VASP_CMD", VASP_CMD)
+    db_file = c.get("DB_FILE", DB_FILE)
+    user_incar_settings = c.get("USER_INCAR_SETTINGS")
+    half_kpts = c.get("HALF_KPOINTS_FIRST_RELAX",HALF_KPOINTS_FIRST_RELAX)
+    ediffg = user_incar_settings.get("EDIFFG",-0.05) if user_incar_settings else -0.05
+
+    wf = get_wf(structure, "optimize_only.yaml",
+                vis=MVLScanRelaxSet(structure, user_incar_settings=user_incar_settings),
+                common_params={"vasp_cmd": vasp_cmd, "db_file": db_file})
+
+    wf = use_custodian(wf,custodian_params={"ediffg":ediffg,
+                                            "max_force_threshold":0,
+                                            "half_kpts_first_relax":half_kpts,
+                                            "job_type":"metagga_relaxation_run",
+                                            "db_file":db_file,
+                                            "vasp_cmd":vasp_cmd})
+    wf = add_common_powerups(wf, c)
+
+    if c.get("ADD_WF_METADATA", ADD_WF_METADATA):
+        wf = add_wf_metadata(wf, structure)
+
+    if not c.get("KEEP_WAVECAR", False):
```

Comment:
I'd like for workflows to try to keep the same sorts of "defaults" to prevent surprises and inconsistencies when transitioning. e.g., going from PBE to SCAN should just change PBE to SCAN, not start adding in other logic and preferences by default. Also, I prefer that workflows be as "plain" as possible unless the user requests otherwise. Otherwise I end up with a whole lot of help tickets as to why they ran 50 workflows with atomate but realized the WAVECAR doesn't seem to be retained.

Thus you should:
* rename/invert KEEP_WAVECAR to REMOVE_WAVECAR (the latter is more of the active step you are taking)
* default REMOVE_WAVECAR to False, since this is "least surprise"
* allow a config.py option to have REMOVE_WAVECAR always be True in case someone wants to take the active step of removing WAVECARS and don't want to set the parameter each time
* copy-paste the same code into all preset workflows where WAVECAR is likely unimportant

Reply:
Noramally, the user sets LWAVE to true and gets a WAVECAR when they want. With the preconditioning step, custodian leaves behind several copies of WAVECARs from the original GGA opt. This is there to clean that up.  I think I may be able to integrate this into custodian now that I think about it.